src/main/java/com/github/princesslana/eriscasper/rest/AuditLogRoute.java

```diff
@@ -0,0 +1,37 @@
+package com.github.princesslana.eriscasper.rest;
+
+import com.github.princesslana.eriscasper.data.Snowflake;
+import com.github.princesslana.eriscasper.data.resource.AuditLogObject;
+import com.github.princesslana.eriscasper.data.resource.Guild;
+import com.github.princesslana.eriscasper.rest.auditlog.GetGuildAuditLogRequest;
+
+public class AuditLogRoute {
+  // https://discordapp.com/developers/docs/resources/audit-log#get-guild-audit-log
```

Comment:
Should this be here?

Reply:
Doesn't have to be, I'll remove it. I think I added it in for myself since I was coming back to it just due to the parsing complexity.